eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Translation.hs

```diff
@@ -12,17 +12,14 @@ where
 import Cardano.Binary
   ( ToCBOR (..),
   )
+-- import Allegra EraGen instance
```

Comment:
is this that helpful? :thinking:

Reply:
Yeah that is useless :smile: I think it was accidental copy and paste at some point. I didn't even notice it. Looks like ormolu moved it together with the changed import. I'll get rid of it